PythonScript/src/HelpController.cpp

```diff
@@ -14,46 +14,50 @@ HelpController::HelpController(HWND hNotepad, HWND hScintilla)
 
 void HelpController::callHelp()
 {
-	//::HtmlHelp(m_hNotepad, getFilename(), HH_DISPLAY_TOPIC, getTopicUrl().c_str());
-	std::string helpFile = getFilename();
-	bool useWeb = false;
+	std::wstring helpFile = getFilename();
 	if (helpFile.empty())
 	{
-		helpFile = "http://npppythonscript.sourceforge.net/docs/";
-		helpFile.append(PYSCR_VERSION_STRING);
-		helpFile.append("/");
-		useWeb = true;
-	}
-
-	if (useWeb)
-	{
-		helpFile.append(getTopicUrl());
-		::ShellExecuteA(m_hNotepad, "open", helpFile.c_str(), NULL, NULL, SW_SHOWDEFAULT);
+		helpFile = L"http://npppythonscript.sourceforge.net/docs/";
+		helpFile.append(WcharMbcsConverter::char2tchar(PYSCR_VERSION_STRING).get());
+		::ShellExecute(m_hNotepad, L"open", helpFile.c_str(), NULL, NULL, SW_SHOWNORMAL);

```

Comment:
appending of the topic is not done here. Why?

Reply:
This is part of the old (1.0.8) online help which can't be used anyway as it isn't setup for
versions > 1.0.8, therefore I haven't done anything.

Reply:
@ClaudiaFrank Could this PR be merged now? Or is there something open? Could you please extend the description what else is fixed apart from #15.